### PR TITLE
Finished proofreading Mage Tower & miscellaneous fixes

### PR DESCRIPTION
**Pull request by Sunaries**
Fixes:
- Corrected misspellings and grammar;
- Removed unnecessary periods in quest tracker;
- Renamed "Carver Tesla" to "Sculptor Tesla" - In the end, a sculptor can also specialize in wood carving;
- Renamed various NPCs;
- Updated some recipe and item names;
- Minor changes to the today tips;
- Proofread Mage Tower and a little bit of Owl Burial Ground. 

@imcgames 

I have a question.

> 3명의 마왕과 그들을 모시는 마군주들.
> 그리고 그 모든 것을 거느리는 존재의 이름은 길티네입니다.
> 
> There are three demon kings with many demon lords to serve them.
> Giltine is the one that rules over them all.

Who is the 3rd? 
What would be the best way to translate this line in terms of the title the superior demons hold? We know that Gesti cannot be a Demon King, but a Demon Queen since she is a female. Another Demon King is "바이가" (Vaiga?). Is this a male or female? And if I know if the 3rd is a female or male, I could maybe change the line to:

"There are 2 demon kings and 1 queen with many demon lords to serve them" for example.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -8870,21 +8870,21 @@ ETC_20150317_008870	Resistance Tip
 ETC_20150317_008871	Attacking with countering properties on monsters can cause more damage.
 ETC_20150317_008872	Archer Tip
 ETC_20150317_008873	Press and hold the attack button to continuously attack the enemy in target. You can also move while attacking using the arrow keys.
-ETC_20150317_008874	Archer's arrow attacks have maximum range. Higher damage is made when shot from a distance.
-ETC_20150317_008875	The accuracy and evasion increases when an Archer type class levels up.
+ETC_20150317_008874	The archer's arrow attacks have a maximum range. Higher damage can be made when you shoot from a distance.
```
Comment (ttgmichael) [inline ETC.tsv]: An archer's arrow attacks have a maximum range.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -980,7 +980,7 @@ QUEST_LV_0100_20150317_000975	{memo X}I hope that crazy magician, Antares, won't
 QUEST_LV_0100_20150317_000976	{memo X}He set a trap so well.{nl}I didn't like him even he was concious.
 QUEST_LV_0100_20150317_000977	{memo X}We have to be careful from here.{nl}This is the place where Antares, a dangerous magician, was imprisoned.{nl}We should move up to the next floor quickly.
 QUEST_LV_0100_20150317_000978	{memo X}He was a great magician before.{nl}But, his ideology was too dangerous.
-QUEST_LV_0100_20150317_000979	Damn, why now..{nl}
+QUEST_LV_0100_20150317_000979	Oh my goodness, why now..{nl}
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:983]: Darn, why now.. ?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -1053,12 +1053,12 @@ QUEST_LV_0100_20150317_001048	$As long as there are Eyes of Surveillance, I can'
...
 QUEST_LV_0100_20150317_001050	{memo X}I was sealed when my sprit was ripped off by a high class demon.{nl}Please defeat Infroroktors for me. I promise I will not be a burden to you.
 QUEST_LV_0100_20150317_001051	{memo X}Eyes of Surveillance are attached to Infroroktor.
-QUEST_LV_0100_20150317_001052	$It's way better without the watchers.
+QUEST_LV_0100_20150317_001052	$This is way better without the watchers.
 QUEST_LV_0100_20150317_001053	{memo X}Those pieces of another me lost the voices to call you.{nl}Please find my voice again from Fallenlem.
 QUEST_LV_0100_20150317_001054	His creations should have voices to ask with{nl}
 QUEST_LV_0100_20150317_001055	$The pieces from my other self are able to call you now.{nl}Now, I can be released.
-QUEST_LV_0100_20150317_001056	I am tied to restraints. The old microbes are threatening me.{nl}Teach them what the manner is.
-QUEST_LV_0100_20150317_001057	You should give them a lesson so that they behave properly.
+QUEST_LV_0100_20150317_001056	My body is restrained. The old microbes are threatening me.{nl}Please teach them some manners.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:1060]: old microbes?

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -3107,7 +3107,7 @@ QUEST_LV_0100_20150317_003102	I will go meet Laimondas
 QUEST_LV_0100_20150317_003103	Church Gate (1)
 QUEST_LV_0100_20150317_003104	Church Gate (2)
 QUEST_LV_0100_20150317_003105	I'll open the front door
-QUEST_LV_0100_20150317_003106	To entrance of Kateen Forest
+QUEST_LV_0100_20150317_003106	To Entrance of Kateen Forest
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:3110]: To the Entrance of Kateen Forest

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -5051,9 +5051,9 @@ QUEST_LV_0100_20150317_005046	The Essence of Light is ready. Talk to Follower Va
 QUEST_LV_0100_20150317_005047	Open the entrance of the chapel
 QUEST_LV_0100_20150317_005048	Vaidutis told you to use the Light Crystal to release the demonic barrier at the chapel gates. Powerful monsters will be attracted to the power of the Light Crystal and may attack you, so he told you to be careful.
 QUEST_LV_0100_20150317_005049	Defeat Mummyghast
-QUEST_LV_0100_20150317_005050	Move to the entrance of Kateen Forest
+QUEST_LV_0100_20150317_005050	Move to Entrance of Kateen Forest
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:5054]: Move to the Entrance of Kateen Forest?

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (Sunaries) [inline QUEST_LV_0100.tsv:983]: The original is more like "Oh my god!", as she is surprised that Antares is free. I wanted to change it to "Oh my Goddess!", but no idea if that would work. :yum:

File: QUEST_LV_0100.tsv (hunk 4 above)
Comment (Sunaries) [inline QUEST_LV_0100.tsv:3110]: Entrance of Kateen Forest is a map name.

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:983]: Oh ok xD

File: QUEST_LV_0100.tsv (hunk 5 above)
Comment (Sunaries) [inline QUEST_LV_0100.tsv:5054]: Entrance of Kateen Forest is a map name. This appears when you are in a field nearby this map. You'll see the big green arrow that would say "Move/Go to Entrance of Kateen Forest".

File: QUEST_LV_0100.tsv (hunk 4 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:3110]: I know! But but English grammar xD
We do say Go to the White House. not Go to White House. :P
Or Go to the Royal Mausoleum

File: QUEST_LV_0100.tsv (hunk 5 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:5054]: Ah, I guess we'll have to leave some "artifacts" if the map is named that way. ><;;

No idea why it couldn't have just been Kateen Forest :P.

Comment (Sunaries) [inline QUEST_LV_0100.tsv:5054]: That's just according to the wiki. The map name could also be named "Kateen Forest Entrance".

Comment (ttgmichael) [inline QUEST_LV_0100.tsv:5054]: Did we do this with the Royal Mausoleum as well? Was it:

Move to the Royal Mausoleum?
or
Move to Royal Mausoleum?

Comment (Sunaries) [inline QUEST_LV_0100.tsv:5054]: I'm not even sure if that line exists. :open_mouth:

**Comment by Sunaries** [comment]
@imcgames Good to merge as far as I know

**Comment by terrapin47** [comment]
:+1: Good to merge

**Comment by imcgames** [comment]
The third would be 'Kartas' and from the three, Gesti is the only female

**Comment by Sunaries** [comment]
@imcgames Thanks. I've updated it to 'There are two demon kings and one demon queen with many demon lords to serve them.'. 

Good to merge.

**Comment by Sunaries** [comment]
@imcgames Thanks. 
I've updated it to 'There are two demon kings and one demon queen with many demon lords to serve them.'.

Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**